README.rst

```diff
@@ -19,33 +19,75 @@ To install from PyPi
 
     $ pip install signalfx_lambda
 
+Auto-Instrumentation
+~~~~~~~~~~~~~~~~~~~~
+
+If the Python auto-instrumentation libraries are available to the wrapper, the
+Lambda handler will be auto-instrumented.
+
+To install the supported instrumentation libraries:
+
+::
+
+    $ pip install signalfx-tracing
+
+    $ sfx-py-trace-bootstrap -t <deployment_package_path>
+
+For more information, check out the full :new-page:`instructions <https://github.com/signalfx/signalfx-python-tracing#installation-and-configuration>`.
+
 Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-    SIGNALFX_AUTH_TOKEN=access token
+    SIGNALFX_ACCESS_TOKEN=access token
+
+    # endpoint for both metrics and tracer. Overridden by SIGNALFX_METRICS_URL
+    # and SIGNALFX_TRACING_URL if set
+    SIGNALFX_ENDPOINT_URL=endpoint url
 
-    # optional
+    # optional metrics and tracing configuration
 
-    SIGNALFX_INGEST_ENDPOINT=ingest endpoint [ default: https://pops.signalfx.com ]
+    SIGNALFX_METRICS_URL=ingest endpoint [ default: https://pops.signalfx.com ]
     SIGNALFX_SEND_TIMEOUT=timeout in seconds for sending datapoint [ default: 0.3 ]
 
+    SIGNALFX_TRACING_URL=tracing endpoint [ default: https://ingest.signalfx.com/v1/trace ]
+
+SIGNALFX_TRACING_URL can be used to configure a common endpoint for metrics and
+traces, as is the case when forwarding with the Smart Gateway.
+
+If either SIGNALFX_TRACING_URL or SIGNALFX_METRICS_URL are set, they will take
+precendence over SIGNALFX_TRACING_URL for their respective components.
+
 Wrapping a function
 ~~~~~~~~~~~~~~~~~~~
 
-Decorate your handler with @signalfx_lambda.wrapper
+There are two wrappers provided.
+
+For metrics, decorate your handler with @signalfx_lambda.metrics_wrapper
+
+::
+
+    import signalfx_lambda
+
+    @signalfx_lambda.metrics_wrapper
+    def handler(event, context):
+        # your code
+
+For tracing, use the @signalfx_lambda.tracing_wrapper decorator
 
 ::
 
     import signalfx_lambda
 
-    @signalfx_lambda.wrapper
+    @signalfx_lambda.tracing_wrapper
```

Comment:
This appears to be another breaking change that could be prevented.

imo, decorators are assumed to be wrappers, so I'd prefer something like `@signalfx_lambda.metric` being the same as the original `wrapper` (which would remain usable) and `@signalfx_lambda.trace` bringing the new functionality.